health-services/project-factory/src/server/utils/sheetManageUtils.ts

```
@@ -0,0 +1,356 @@
+            code: error.code,
+            description: error.description,
+            message: error.message
+        }
+    }
+    await produceModifiedMessages({ generatedResource: [responseToSend] }, config?.kafka?.KAFKA_UPDATE_GENERATED_RESOURCE_DETAILS_TOPIC);
+}
+
+async function createBasicTemplateViaConfig(responseToSend:any,templateConfig: any, localizationMap: any) {
+    const newWorkbook = new ExcelJS.Workbook();
+    const tenantId = responseToSend?.tenantId;
+    for(const sheet of templateConfig?.sheets) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)